README.md

```diff
@@ -58,4 +103,61 @@ An example of a **meaningful** change:
 - `falsepos-...` files are cases any filter should catch
 - `truepos...` files are cases of changes we care about
 
-Filters will need to be tested (and possibly trained) against real datasets at some point
+This is a small but illustrative sample. Many more samples will be made
+available as soon as possible.
+
+### Schema
+
+This describes the schema of the SQL databases shared by the Rails app in
+web-monitoring-db and the Python processing bakend in web-monitoring-processing.
+Review the Definition of Terms section above, which corresponds to these tables.
+
+Every table includes:
+
+* uuid: UUID4 unique indentifier
+* created_at: internal detail of the database
+* updated_at: internal detail of the database
+
+in addition to the table-specific fields listed below.
+
+#### Pages
+
+* url: URL, which may be updated over time if a page is moved
+* title: `<title>` tag
+* agency: Governemnt agency
+* site: Goverment site (?)
+
+#### Versions
+
+* page_uuid: reference to a Page
+* capture_time: when this snapshot of the Page was acquired
+* uri: path to stored (HTML) data; could be a filepath, S3 bucket, etc.
+* version_hash: sha256 hash of stored data
+* source_type: name of source (such as 'Internet Archive')
```

Comment:
When I finish https://github.com/edgi-govdata-archiving/web-monitoring-db/pull/15, I’ll try and update this with current used values (only `versionista` for now) and the expected schema for `source_metadata` given those values. (Also please bug me if I forget and fail to do so within a day.)